skimage/color/colorconv.py

```diff
@@ -56,7 +56,9 @@
 from warnings import warn
 import numpy as np
 from scipy import linalg
-from ..util import dtype, dtype_limits
+from ..util import dtype, dtype_limits, img_as_ubyte
+from matplotlib import pyplot as plt
```

Comment:
@JDWarner "standard" form? I've seen both and prefer the current version.

As to try/except, I'd forgotten that MPL is not a requirement. In that case, I would suggest moving the import to a try: except inside the function call with an informative error message.